setup.py

```diff
@@ -1,15 +1,15 @@
 from setuptools import setup, find_packages
 
 
-with open('README.md') as f:
+with open('PACKAGE_README.md') as f:
     readme = f.read()
```

Comment:
pip doesn't happy with some format in README.md, it throw error when upload
https://pypi.org/help/#description-content-type